libs/domain/search/facet-rating/stories/static/static.stories.ts

```diff
@@ -0,0 +1,55 @@
+import { resolve } from '@spryker-oryx/di';
+import { MockRouterService } from '@spryker-oryx/experience/mocks';
+import { RouterService } from '@spryker-oryx/router';
+import { FacetListService } from '@spryker-oryx/search';
+import { Story } from '@storybook/web-components';
+import { TemplateResult, html } from 'lit';
+import { storybookPrefix } from '../../../.constants';
+
+export default {
+  title: `${storybookPrefix}/Facet Rating`,
+};
+
+const Template: Story = (): TemplateResult => {
+  resolve(FacetListService);
+
+  const router = resolve(RouterService) as unknown as MockRouterService;
+  router.params$.next({});
+
+  return html`<oryx-search-facet-rating
+      name="Rating"
+      open
+    ></oryx-search-facet-rating>
+    <script>
```

Comment:
use [storybook play](https://storybook.js.org/docs/6.5/web-components/writing-stories/play-function) instead of this